nixos/tests/fstab.nix

```diff
@@ -0,0 +1,48 @@
+import ./make-test.nix ({ pkgs, lib, ... }:
```

Comment:
Do you think we need an extra test for that? I don't see how this would break in future?

Reply:
Fair enough. Take it or leave it I guess...

Reply:
We already have a zfs test, if it was integrate there it might be ok. Otherwise we probably don't want to maintain both.

Reply:
Not zfs specific and you make good point so I'll just drop the test.